test/use-swr-integration.test.tsx

```diff
@@ -311,6 +311,46 @@ describe('useSWR', () => {
     )
   })
 
+  it('should revalidate even if initialData is provided', async () => {
+    const fetcher = async key => {
+      await sleep(50)
+      return key
+    }
+
+    function Page() {
+      const [key, setKey] = useState('initial-data-with-initial-data')
+      const { data } = useSWR(key, fetcher, {
+        initialData: 'Initial'
+      })
+      return (
+        <div onClick={() => setKey('initial-data-with-initial-data-update')}>
+          {data ? `hello, ${data}` : 'loading'}
+        </div>
+      )
+    }
+
+    const { container } = render(<Page />)
+
+    // render with the initial data
+    await screen.findByText('hello, Initial')
+
+    await act(() => sleep(1))
+    fireEvent.focus(window)
+
+    await screen.findByText('hello, initial-data-with-initial-data')
+
+    // change the key
+    await act(() => sleep(1))
+    fireEvent.click(container.firstElementChild)
+
+    // a request is still in flight
+    await act(() => sleep(10))
+    expect(container.firstChild.textContent).toMatchInlineSnapshot(`"loading"`)
```

Comment:
I'm not sure what the expected value is.
But I expected SWR returns `undefined` as the return value of `data` when revalidating. But SWR returns the `initialData` Is it an expected behavior?

Reply:
Yeah `initialData` works like a fallback value currently. I think we need to better name for it, maybe just `fallback`. And for the real `initialData`, we need a completely new API, which works like a prefill. (also cc @huozhi)

Reply:
That seems to be great 👍  `fallback` would be more intuitive than `initialData` for the behavior.